packages/utils/src/chainflip.ts

```diff
@@ -88,139 +100,156 @@ export function readAssetValue<T>(
     rpcAsset: 'DOT',
     decimals: 10,
   },
   Btc: {
     chain: 'Bitcoin',
+    symbol: 'BTC',
     rpcAsset: 'BTC',
     decimals: 8,
   },
   ArbUsdc: {
     chain: 'Arbitrum',
+    symbol: 'USDC',
     rpcAsset: 'USDC',
     decimals: 6,
   },
   ArbEth: {
     chain: 'Arbitrum',
+    symbol: 'ETH',
     rpcAsset: 'ETH',
     decimals: 18,
   },
   Sol: {
     chain: 'Solana',
+    symbol: 'SOL',
     rpcAsset: 'SOL',
     decimals: 9,
   },
   SolUsdc: {
     chain: 'Solana',
+    symbol: 'USDC',
     rpcAsset: 'USDC',
     decimals: 6,
   },
   HubDot: {
     chain: 'Assethub',
+    symbol: 'DOT',
     rpcAsset: 'DOT',
     decimals: 10,
   },
   HubUsdc: {
     chain: 'Assethub',
+    symbol: 'USDC',
     rpcAsset: 'USDC',
     decimals: 6,
   },
   HubUsdt: {
     chain: 'Assethub',
+    symbol: 'USDT',
     rpcAsset: 'USDT',
     decimals: 6,
   },
-} as const satisfies Record<
-  ChainflipAsset,
-  {
-    chain: ChainflipChain;
-    rpcAsset: RpcAsset;
-    decimals: number;
-  }
->;
+} as const satisfies InternalAssetMap<{
+  chain: ChainflipChain;
+  symbol: AssetSymbol;
+  /** @deprecated use `symbol` instead */
+  rpcAsset: AssetSymbol;
+  decimals: number;
+}>;
 
 export const chainConstants = {
   Ethereum: {
-    assets: ['Eth', 'Flip', 'Usdc', 'Usdt'],
+    chainflipAssets: ['Eth', 'Flip', 'Usdc', 'Usdt'],
+    assets: ['ETH', 'FLIP', 'USDC', 'USDT'],
     rpcAssets: ['ETH', 'FLIP', 'USDC', 'USDT'],
     gasAsset: 'Eth',
     addressType: 'Eth',
     blockTimeSeconds: 12,
   },
   Polkadot: {
-    assets: ['Dot'],
+    chainflipAssets: ['Dot'],
+    assets: ['DOT'],
     rpcAssets: ['DOT'],
     gasAsset: 'Dot',
     addressType: 'Dot',
     blockTimeSeconds: 6,
   },
   Bitcoin: {
-    assets: ['Btc'],
+    chainflipAssets: ['Btc'],
+    assets: ['BTC'],
     rpcAssets: ['BTC'],
     gasAsset: 'Btc',
     addressType: 'Btc',
     blockTimeSeconds: 10 * 60,
   },
   Arbitrum: {
-    assets: ['ArbUsdc', 'ArbEth'],
+    chainflipAssets: ['ArbUsdc', 'ArbEth'],
+    assets: ['USDC', 'ETH'],
     rpcAssets: ['USDC', 'ETH'],
     gasAsset: 'ArbEth',
     addressType: 'Arb',
     blockTimeSeconds: 0.26,
   },
   Solana: {
-    assets: ['Sol', 'SolUsdc'],
+    chainflipAssets: ['Sol', 'SolUsdc'],
+    assets: ['SOL', 'USDC'],
     rpcAssets: ['SOL', 'USDC'],
     gasAsset: 'Sol',
     addressType: 'Sol',
     blockTimeSeconds: 0.8,
   },
   Assethub: {
-    assets: ['HubDot', 'HubUsdt', 'HubUsdc'],
+    chainflipAssets: ['HubDot', 'HubUsdt', 'HubUsdc'],
+    assets: ['DOT', 'USDT', 'USDC'],
     rpcAssets: ['DOT', 'USDT', 'USDC'],
     gasAsset: 'HubDot',
     addressType: 'Hub',
     blockTimeSeconds: 12,
   },
-} as const satisfies Record<
-  ChainflipChain,
-  {
-    assets: ChainflipAsset[];
-    rpcAssets: RpcAsset[];
-    gasAsset: ChainflipAsset;
-    addressType: AddressType;
-    blockTimeSeconds: number;
-  }
->;
+} as const satisfies ChainMap<{
+  chainflipAssets: ChainflipAsset[];
+  assets: AssetSymbol[];
```

Comment:
for consistency maybe
```suggestion
  assetSymbols: AssetSymbol[];
```

Reply:
Yeah I had that at first then changed it to this, think it's slightly better